app/config.js

```diff
@@ -4,4 +4,7 @@ export default {
     MAX_ALLOWED_UNUSED_ADDRESSES: 20,
     TRANSACTION_REQUEST_SIZE: 20
   },
+  trezor: {
+    DEFAULT_CARDANO_PATH: 'm/44\'/1815\'/0\''
```

Comment:
maybe a comment to the bip that specifies this?

Reply:
yes,
added: 
// https://github.com/trezor/connect/blob/develop/docs/methods/cardanoGetPublicKey.md
// https://cardanolaunch.com/assets/Ed25519_BIP.pdf